python/paddle/device.py

```diff
@@ -13,10 +13,25 @@
 # limitations under the License.
 
 # TODO: define the functions to manipulate devices 
-# __all__ = ['cpu_places',
-#            'CPUPlace',
-#            'cuda_pinned_places',
-#            'cuda_places',
-#            'CUDAPinnedPlace',
-#            'CUDAPlace',
-#            'is_compiled_with_cuda']
+from paddle.fluid import core
+__all__ = [
+    'cudnn_version',
+    #            'cpu_places',
+    #            'CPUPlace',
+    #            'cuda_pinned_places',
+    #            'cuda_places',
+    #            'CUDAPinnedPlace',
+    #            'CUDAPlace',
+    #            'is_compiled_with_cuda'
+]
+
+
+def cudnn_version():
```

Comment:
export a variable instead? e.g.,

```python
cudnn_version = _cudnn_version()
```

Reply:
this fucntion will be exposed to user?

Reply:
scratch that, python does not have `@property` mechanism for module